### PR TITLE
Unittesting helper.py and enum.py

### PR DESCRIPTION
**Pull request by rayklauck**
Reused some of the old tests

### CODE REVIEW

**Review by tobiasah — CHANGES_REQUESTED** [review]
🌞 next time, clean up your commit messages before requesting a review

File: tests/nodetree/test_entry_point.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    session_mock.list_nodes_info = Mock(return_value=_get_future(list_node_info))
+
+    nodetree_mock = Mock()
+    nodetree_mock.construct_nodetree = Mock(return_value="result")
+
+    # TODO __new__ mock breaks subsequest tests (pytest bug)... How to work around?
+    with patch(
+        "labone.nodetree.node.NodeTreeManager.__new__",
+        return_value=nodetree_mock,
+    ) as new_mock, patch(
```
[the remark this answers is not in the document]
Comment (rayklauck) [reply tests/nodetree/test_entry_point.py:43]: I am not sure in which manner one could do this here

File: tests/nodetree/test_enum.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    parsed = parser(
+        AnnotatedValue(path="/zi/mds/groups/0/status", value=1),
+    )
+    assert isinstance(parsed.value, Enum)
+    assert parsed.value.value == 1  # IntEnum equality
+    assert parsed.value.name == "Sync"
+
+
+def test_default_parser_value_not_in_enum(parser):
+    parsed = parser(
```
[the remark this answers is not in the document]
Comment (rayklauck) [reply tests/nodetree/test_enum.py:105]: There is no warning thrown and checking it lets the test fail. Is logger.warning really throwing a warning and not just logging one?

[the remark this answers is not in the document]
Comment (tobiasah) [reply tests/nodetree/test_enum.py]: ok than never mind

**Review by tobiasah — APPROVED** [review]

**Pull request merged**